File: Python/PRAK204-2210817310009-Mario Franca Wijaya.py
print("INPUT")
jari = float(input("Masukan Nilai jari-jari\t:"))
tinggi = float(input("Masukan Nilai tinggi\t:"))
volume = 22 * tinggi * (jari * jari) /7 
luas = 2 * 22 * jari * (jari + tinggi) / 7
keliling = 2 * 22 * jari / 7
print("OUTPUT")
print("Volume = {:.2f} cm".format(volume))
print("Luas = {:.2f} cm".format(luas))
print("Keliling = {:.2f} cm".format(keliling))

File: C/PRAK204-2210817310009-MarioFrancaWijaya-.c
#include <stdio.h>

int main(){
    float jari, tinggi, volume, luas, keliling; 
    
    printf("Masukan nilai jari-jari\t:");
    scanf("%f", &jari);
    printf("Masukan nilai tinggi\t:");
    scanf("%f", &tinggi);

    volume = 22 * tinggi * (jari * jari) /7;
    luas = 2 * 22 * jari * (jari + tinggi) / 7;
    keliling = 2 * 22 * jari / 7;

    printf("OUTPUT\n");
    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f", keliling);

    return 0;
}